### ISSUE TITLE
zero dice during the training

### ISSUE THREAD

**Issue by lc82111** [wentaozhu/AnatomyNet-for-anatomical-segmentation#6]
Dear Author:
During the training of baselineSERes18Conc.py, I found the dice is almost always to be 0 for all the classes. Is It normal? I used the data you provide at Google Drive and all the original hyper-parameters.
```
epoch 47 TRAIN loss 8.3984
test loss 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000
best test loss 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000, 0.0000
```

**Comment by wentaozhu** [comment]
Thank you very much for your interest! How about keeping nn.BatchNorm3d instead of #nn.BatchNorm3d(planes))? I commented it after the upload. You can find the details in https://github.com/wentaozhu/AnatomyNet-for-anatomical-segmentation/commit/ecbc4a673fb4b9e93a8305cbba69552fb425520b Tell me if it works. Thanks!

**Comment by wentaozhu** [comment]
Did you solve the problem?

**Comment by lc82111** [comment]
I would try it this afternoon, thank you!

**Comment by lc82111** [comment]
It works ~. However, I wonder why such a minor change has a significant impact on the training.

**Comment by wentaozhu** [comment]
If we do not use the batchnorm, we should set bias as true. Anyway, congratulations and good luck!

**Comment by wentaozhu** [comment]
Now you can download or clone the latest version. Thank you very much for the interest!

**Comment by lc82111** [comment]
> If we do not use the batchnorm, we should set bias as true. Anyway, congratulations and good luck!

It makes sense. thank you.
